### PR TITLE
85 document all libraries currently in use

### PR DESCRIPTION
**Pull request by Keyshal-211**

### CODE REVIEW

File: dev-manuals/backend/Libraries/AllService_libraries.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+| **Course Service**     | JSR 305                              | 3.0.2     | Removes a Gradle warning about an unknown annotation.                |
+
+| Service                | Library                              | Version   | Purpose                                                              |
+|:-----------------------|:-------------------------------------|:----------|:---------------------------------------------------------------------|
+| **Reward Service**     | Content Service                      | 0.1.2     | Includes content service for DTOs.                                   |
+
+| Service                                                           | Library                              | Version   | Purpose                                                              |
+|:------------------------------------------------------------------|:-------------------------------------|:----------|:---------------------------------------------------------------------|
+| **Course Service**, **Reward Service**, **SkillLevel Service**    | Spring Boot Starter Webflux          |           | Provides dependencies for building reactive web applications.         |
```
Comment (PaulBredl) [inline dev-manuals/backend/Libraries/AllService_libraries.md:52]: That is a bit vague. We use it in Reward Service and Skilllevel service for inter-service communication, you could add that

File: dev-manuals/backend/Libraries/AllService_libraries.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+| **Content Service**    | H2 Database                          |           | In-memory database for local development and testing.                |
+
+| Service                | Library                              | Version   | Purpose                                                              |
+|:-----------------------|:-------------------------------------|:----------|:---------------------------------------------------------------------|
+| **Course Service**     | JSR 305                              | 3.0.2     | Removes a Gradle warning about an unknown annotation.                |
+
+| Service                | Library                              | Version   | Purpose                                                              |
+|:-----------------------|:-------------------------------------|:----------|:---------------------------------------------------------------------|
+| **Reward Service**     | Content Service                      | 0.1.2     | Includes content service for DTOs.                                   |
```
Comment (PaulBredl) [inline dev-manuals/backend/Libraries/AllService_libraries.md:48]: Doesn't the skill level service also contain the content service as dependeny?

**Comment by PaulBredl** [comment]
Could you also provide links to the libraries?

File: dev-manuals/backend/Libraries/AllService_libraries.md (hunk 2 above)
Comment (Keyshal-211) [reply dev-manuals/backend/Libraries/AllService_libraries.md:48]: yes, I forgot sorry, I added

File: dev-manuals/backend/Libraries/AllService_libraries.md (hunk 1 above)
Comment (Keyshal-211) [reply dev-manuals/backend/Libraries/AllService_libraries.md:52]: done

**Comment by PaulBredl** [comment]
why did you close the PR?

**Pull request closed without merging**